### PR TITLE
(370) Exporting lost beds reporting

### PR DESCRIPTION
**Pull request by pezholio**
This adds a page for workflow managers to download a report for lost beds within a given month in Excel format via the API. The design is pretty bare bones, but does the trick.

## Screenshots

image

image

### CODE REVIEW

**Comment by pezholio** [comment]
[Exporting lost beds reporting](https://trello.com/c/05PzcEFq/370-exporting-lost-beds-reporting)

File: server/utils/dateUtils.ts [hunk 1]
```diff
@@ -206,4 +206,30 @@ export const dateIsInThePast = (dateString: string): boolean => {
   return isPast(date)
 }
 
+export const monthOptions = [
```
Comment (bryangaledxw) [inline server/utils/dateUtils.ts:209]: Really like this!

File: integration_tests/pages/admin/reportPage.ts [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  constructor() {
+    super('Reports')
+  }
+
+  static visit(): ReportPage {
+    cy.visit(paths.admin.reports.new({}))
+    return new ReportPage()
+  }
+
+  expectDownload(): void {
```
Comment (bryangaledxw) [inline integration_tests/pages/admin/reportPage.ts:14]: My instinct would be to push this down into the base Page class

**Review by bryangaledxw — APPROVED** [review]
LGTM

**Pull request merged**